src/test_utils.rs

```diff
@@ -400,11 +400,11 @@ impl TestContext {
     /// Sets a name for this [`TestContext`] if one isn't yet set.
     ///
     /// This will show up in events logged in the test output.
-    pub fn set_name(&self, name: impl Into<String>) {
+    pub fn set_name(&self, name: &str) {
         let mut context_names = CONTEXT_NAMES.write().unwrap();
         context_names
             .entry(self.ctx.get_id())
-            .or_insert_with(|| name.into());
+            .or_insert_with(|| name.to_string());
```

Comment:
Seems it's better to take a `String` parameter then. At least this way temporary `String` objects can be passed to the function for free

Reply:
None of the callers of this function actually has a `String` at hand, all of them would just need to call `to_string()` themselves. Also, this is test code, where performance is dominated by the compile times, not by the time the tests actually need to run.